src/kfmon.h

```diff
@@ -5,8 +5,8 @@ extern "C" {
 #endif
 
 #include <stddef.h>
-#include <stdlib.h>
 #include <stdint.h>
+#include <stdlib.h>
 #include "action.h"
```

Comment:
Hopefully other files are in alphabetical order, too (I've been doing it manually, but I might have missed it in one or two of the recent commits).

Reply:
I may have attempted to do so semi-consciously in other places, but I wouldn't bet my life on it ^^.